package.json

```diff
@@ -36,8 +36,11 @@
     "css-loader": "^0.21.0",
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.9.0",
+    "node-sass": "^3.4.1",
+    "normalize.css": "^3.0.3",
```

Comment:
Normalize should come as a dependency from Toolbox. The `commons.scss` includes normalize, it's failing the build without it for you? `node-sass` is ok there I think